ios/TPSStripe/TPSCardField.m

```diff
@@ -179,4 +179,18 @@ - (void)paymentCardTextFieldDidChange:(STPPaymentCardTextField *)textField {
                 });
 }
 
+- (void)paymentCardTextFieldDidEndEditing:(STPPaymentCardTextField *)textField {
+    if (!_onEndEditing) {
+        return;
+    }
+    _onEndEditing(@{
```

Comment:
I'm interested in opinions about what makes sense to pass through as arguments to this callback.